### PR TITLE
FuelSavingsPage-coverage

### PR DESCRIPTION
**Pull request by mattiaerre**
# Description

- use enzyme mount to test actions

- use jest to snapshot connected component

### CODE REVIEW

**Comment by mattiaerre** [comment]
@coryhouse this PR fully covers the container component and shows how to use enzyme to simulate user interaction as well as jest snapshot to dump the rendered version of the connected component; this is just an idea but as a boilerplate, it is nice to have examples of different testing strategies;
thanks,
and happy Christmas you all 🎅

File: src/components/containers/FuelSavingsPage.spec.js [hunk 1]
```diff
@@ -1,19 +1,74 @@
...
+      <FuelSavingsPage
+        actions={actions}
+        fuelSavings={initialState.fuelSavings}
+      />
     );
 
     expect(wrapper.find(FuelSavingsForm).length).toEqual(1);
   });
+
+  it("upon clicking save, saveFuelSavings is called", () => {
```
Comment (coryhouse) [inline src/components/containers/FuelSavingsPage.spec.js]: Rename: `calls saveFuelSavings upon clicking save`.

File: src/components/containers/FuelSavingsPage.spec.js [hunk 2]
```diff
@@ -1,19 +1,74 @@
...
+
+    const save = wrapper.find('input[type="submit"]');
+    save.simulate("click");
+
+    expect(actions.saveFuelSavings).toHaveBeenCalledWith(
+      initialState.fuelSavings
+    );
+  });
+
+  it("upon changing a field, calculateFuelSavings is called", () => {
```
Comment (coryhouse) [inline src/components/containers/FuelSavingsPage.spec.js]: Rename `calls calculateFuelSavings upon changing a field`

File: src/components/containers/FuelSavingsPage.spec.js [hunk 3]
```diff
@@ -1,19 +1,74 @@
...
+    input.simulate("change", { target: { name, value } });
+
+    expect(actions.calculateFuelSavings).toHaveBeenCalledWith(
+      initialState.fuelSavings,
+      name,
+      value
+    );
+  });
+
+  it("to match snapshot", () => {
```
Comment (coryhouse) [inline src/components/containers/FuelSavingsPage.spec.js]: Rename `should match snapshot`

**Comment by coryhouse** [comment]
Thanks for the PR! I've provided a few minor rename suggestions.

**Comment by mattiaerre** [comment]
thanks @coryhouse for your feedback; I see, you are using `it` as part of your test description, I like that 😊 let me know if you want me to have a look at other parts to increase the code coverage

**Comment by coryhouse** [comment]
Thanks @mattiaerre! 👍

**Pull request merged**